rails-template.rb

```diff
@@ -0,0 +1,596 @@
+    "exclude": ["**/*.spec.ts", "node_modules", "vendor", "public"],
+    "compileOnSave": false
+  }
+JS
+
+file "package.json", <<~JS
+  {
+    "scripts": {
+      "format": "prettier --write \\"src/**/*.{tsx,ts,scss,json}\\"",
+      "lint": "yarn lint:types && yarn lint:style && yarn lint:format",
+      "lint:types": "tsc --noEmit",
+      "lint:style": "eslint app/javascript/**/*.ts --max-warnings 0",
+      "lint:format": "prettier --list-different \\"app/**/*.{ts,scss,json}\\"",
+      "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds",
+      "bundle-size": "npx source-map-explorer app/assets/builds/application.js app/assets/builds/application.js.map --no-border-checks",
+      "start": "yarn build --watch"
+    },
+    "license": "MIT"
+  }
+JS
+
+file "app/assets/builds/.keep", ""
+
+file ".eslintrc.js", <<~JS
+  module.exports = {
+    env: {
+      browser: true,
+      es6: true,
+    },
+    ignorePatterns: ['node_modules/'],
+    parser: '@typescript-eslint/parser',
+    extends: [
+      'plugin:@typescript-eslint/recommended',
+      'plugin:prettier/recommended',
+    ],
+    parserOptions: {
+      ecmaVersion: 2018,
+      sourceType: 'module',
+    },
+    plugins: ['@typescript-eslint'],
+    rules: {
+      'no-console': 2,
+      '@typescript-eslint/explicit-function-return-type': [
+        'error',
+        {
+          allowExpressions: true,
+        },
+      ],
+      eqeqeq: 2,
+    },
+    settings: {
+      react: {
+        version: 'detect', // Tells eslint-plugin-react to automatically detect the version of React to use
+      },
+    },
+  };
+JS
+
+file "app/javascript/application.ts", <<~JS
+  import '@hotwired/turbo-rails';
+  import { start } from '@nerdgeschoss/shimmer';
+  import { application } from 'controllers/application';
+  import './controllers';
+
+  start({ application });
+JS
+
+file "app/javascript/controllers/application.ts", <<~JS
+  import { Application } from '@hotwired/stimulus';
+
+  const application = Application.start();
+
+  application.debug = false;
+
+  export { application };
+JS
+
+initializer "i18n.rb", <<~RUBY
+  # frozen_string_literal: true
+
+  I18n.default_locale = :en
+  disabled_locales = ENV["DISABLED_LOCALES"].to_s.split(",").map(&:downcase).map(&:to_sym)
+  I18n.available_locales = [:en] - disabled_locales
+  I18n.enforce_available_locales = true
+
+  Rails.application.configure do
+    config.i18n.fallbacks = [:en] - disabled_locales
+  end
+RUBY
+
+initializer "oj.rb", <<~RUBY
+  # frozen_string_literal: true
+
+  Oj.optimize_rails
+RUBY
+
+initializer "serviceworker.rb", <<~RUBY
+  # frozen_string_literal: true
+
+  Rails.application.configure do
+    config.serviceworker.routes.draw do
+      match "/serviceworker.js"
+    end
+  end
+RUBY
+
+initializer "uuid.rb", <<~RUBY
+  # frozen_string_literal: true
+
+  Rails.application.configure do
+    config.generators do |g|
+      g.orm :active_record, primary_key_type: :uuid
```

Comment:
@JensRavens : Don't we want to stick as much as possible to integer IDs to simply things, unless there is a need for UUIDs? Since _Rails_ is constantly making the assumption of sequential IDs for `last`/`first` and APIs of the sort, it would make sense. We are also using those default ID types for _Copa-Coupona_ and _FlipVinyls_.